File: src/main/resources/output/IFC4X3_csharp/Entities/IfcTimeSeries/IfcTimeSeries.cs
public class IfcTimeSeries
{
    public IfcLabel Name { get; set; }
    public IfcText Description { get; set; }
    public IfcDateTime StartTime { get; set; }
    public IfcDateTime EndTime { get; set; }
    public IfcTimeSeriesDataTypeEnum TimeSeriesDataType { get; set; }
    public IfcDataOriginEnum DataOrigin { get; set; }
    public IfcLabel UserDefinedDataOrigin { get; set; }
    public IfcUnit Unit { get; set; }

    // === EXTENDED BY ===
    // IfcIrregularTimeSeries
    // IfcRegularTimeSeries

    // === INVERSE CLAUSES ===
    // HasExternalReference : SET [1:?] OF IfcExternalReferenceRelationship FOR RelatedResourceObjects
}

File: src/main/resources/output/IFC4X3_python/Entities/IfcTimeSeries/IfcTimeSeries.py
class IfcTimeSeries:
    def __init__(self):
        self.Name: IfcLabel = None
        self.Description: IfcText = None
        self.StartTime: IfcDateTime = None
        self.EndTime: IfcDateTime = None
        self.TimeSeriesDataType: IfcTimeSeriesDataTypeEnum = None
        self.DataOrigin: IfcDataOriginEnum = None
        self.UserDefinedDataOrigin: IfcLabel = None
        self.Unit: IfcUnit = None

    # === EXTENDED BY ===
    # IfcIrregularTimeSeries
    # IfcRegularTimeSeries

    # === INVERSE CLAUSES ===
    # HasExternalReference : SET [1:?] OF IfcExternalReferenceRelationship FOR RelatedResourceObjects
